### DISCUSSION TITLE
Http api call limit, how it works?

### DISCUSSION THREAD

**Discussion by limonspb** [community/community#24342] [Apps, API and Webhooks]
<p>Hello!</p>
<p>There is a way to request a repo structure in JSON format like that (example)<br>
<aside class="onebox githubfolder">
  <header class="source">
      <a href="https://api.github.com/repos/betaflight/betaflight/contents/src/main/fc?ref=akk_vtx_fix" target="_blank" rel="noopener nofollow ugc">api.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://api.github.com/repos/betaflight/betaflight/contents/src/main/fc?ref=akk_vtx_fix" target="_blank" rel="noopener nofollow ugc"></a></h3>

<p><a href="https://api.github.com/repos/betaflight/betaflight/contents/src/main/fc?ref=akk_vtx_fix" target="_blank" rel="noopener nofollow ugc">//api.github.com/repos/betaflight/betaflight/contents/src/main/fc</a></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
</p>
<p>How does the limitation work on the number of such “API” calls? I can make these calls without authentification at all from the incognito browser window.</p>
<p>Is it 1000 calls from my IP address per hour? Or is it 1000 calls to this repo per hour total?</p>
<p>Thanks!</p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/limonspb">@limonspb</a> there is some information here on this topic<br>
</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting" target="_blank" rel="noopener nofollow ugc">Resources in the REST API - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by allunderone** [comment]
<p>I’m fairly new too this</p>
